training_gallery.py

```diff
@@ -160,13 +171,17 @@ def train_faces(self):
 
                 image = face_recognition.load_image_file(imagePath)
                 face_locations = face_recognition.face_locations(image)
-                face_encodings = face_recognition.face_encodings(image, face_locations)
+                face_encodings = face_recognition.face_encodings(
+                    image, face_locations)
+
                 for encoding in face_encodings:
                     known_faces.append(encoding)
                     known_names.append(name)
 
-        with open(self.FACES_DAT, 'wb') as f:
-            pickle.dump((known_names, known_faces), f)
+        # with open(self.FACES_DAT, 'wb') as f:
+        #     pickle.dump((known_names, known_faces), f)
+        f = open(self.FACES_DAT, 'wb')
```

Comment:
```suggestion
        with open(self.FACES_DAT, 'wb') as f:
            pickle.dump((known_names, known_faces), f)
        # f = open(self.FACES_DAT, 'wb')
        # pickle.dump((known_names, known_faces), f)
```

parece que nao tava treinando, com isso funciona